lib/fluent/plugin/out_file.rb

```diff
@@ -87,7 +102,13 @@ def configure(conf)
       @formatter = Plugin.new_formatter(@format)
       @formatter.configure(conf)
 
-      @buffer.symlink_path = @symlink_path if @symlink_path
+      if @symlink_path && @buffer.respond_to?(:path)
+        (class << @buffer; self; end).module_eval do
+          prepend SymlinkBufferMixin
```

Comment:
Conflicts may occur if someone want to change L90 (in previous version), but any other changes will not conflict this change.